cert-expiry-finder

```diff
@@ -294,10 +294,29 @@ expiry_after() {
     esac
 }
 
+list_k8s_cert_expiry_days() {
+    local name namespace notafter unixtime delta context
+    if ! which kubectl; then
```

Comment:
Don't forget to test after doing code changes ;)

Also, prefer command -v, which does not require which to exist.

`if ! command -v kubectl 2>/dev/null; then`

Reply:
changed to command -v. what do you mean with "Don't forget to test after doing code changes ;)". if i changed the name to an non existing command it ran as expected